token-swap/js/client/index.ts

```diff
@@ -96,111 +89,6 @@ export const CurveType = Object.freeze({
  * A program to exchange tokens against a pool of liquidity
  */
 export class TokenSwap {
-  /**
```

Comment:
Could we keep all of these declarations and not do the `public ....` in the constructor? There's so many variables that it's useful to have these explanations

Reply:
Most are pointless to be honest, since they repeat the variable name.
Look right above the ctor, the less obvious params are already explained there in a comment.

Reply:
Oh right, in that case, could you document all of the parameters above the constructor?

Reply:
Done